CHANGELOG.md

```diff
@@ -1,5 +1,61 @@
 # [Versions](https://mui.com/versions/)
 
+## v6.1.2
+
+<!-- generated comparing v6.1.1..master -->
+
+_Oct 1, 2024_
+
+A big thanks to the 13 contributors who made this release possible.
+
+### `@mui/material@6.1.2`
+
+- [Autocomplete] Fix listbox opens and closes on click when used with `limitTags` (#42494) @appleSimple
+- [Button] ignore `dark` and `contrastText` if not provided in the theme (#43861) @siriwatknp
+- [Button] Fix regression for color `inherit` (#43862) @siriwatknp
+- [LinearProgress] Fix background color (#43949) @sai6855
+- Support CSS variables with shadow DOM (#43948) @siriwatknp
+- Improve getReactElementRef() utils (#43022) @sai6855
+- [Modal] Replace `show` parameter name with `hide` in modal manager (#43868) @ZeeshanTamboli
+- [Rating] Use Rating `name` as prefix of input element ids (#43829) @yash49
+- [Drawer] Refactor getScrollbarSize usages (#43828) @BrianWoolfolk
+- [Drawer] Fix issue with main window being used instead of iframe's window (#43818) @albarv340
+
+### Docs
+
+- Format Pigment CSS docs (#43812) @oliviertassinari
+- Fix visual bug on dashboard template (#43836) @oliviertassinari
+- Fix pigment-css.md syntax error (#43837) @kdichev
+- Fix Sign-in template form experience (#43838) @oliviertassinari
+- Remove "To be continued" section from v0 –> v1 migration guide (#43832) @samuelsycamore
+- Fix 301 to chromium (#43809) @oliviertassinari
+- [joy-ui] Add missing ComponentLinkHeader components (#43865) @samuelsycamore
+- [Modal] Remove unnecessary type assertion (#43825) @ZeeshanTamboli
+- [table] Stabilize random series in virtualized table demo (#43744) @Janpot
+- [system] Add migration guide link to `@mui/styles` pages (#43833) @samuelsycamore
+
+### Core
```

Comment:
I left these two PRs because they don't seem relevant, but let me know if you think they should be included:

- [website] Improve Next roles section (#43822) @oliviertassinari
- [website] Open the xCharts, eXplore and X general react engineer roles (#43805) @DanailH

Reply:
I would propose that we systematically remove the website https://github.com/mui/material-ui/pull/44069. As a developer who is trying to follow the changes in my dependencies, this is most of the time noise for me. To be fair, I would likely not even read what's in core, I will stop at the npm package changes. It's how I consume Ant Design changelogs at least: https://github.com/ant-design/ant-design/releases/tag/5.21.0 (I used to be subscriber, to their releases, updating all of our relevant GitHub issues after each of their changes).

Reply:
I agree. Website changes don't belong to release notes IMO.